tree/ntuple/v7/doc/specifications.md

```diff
@@ -1,159 +1,560 @@
+| 0x0A |   64 | Int64        | Two's complement, little-endian 8 byte integer                                |
+| 0x0B |   32 | Int32        | Two's complement, little-endian 4 byte integer                                |
+| 0x0C |   16 | Int16        | Two's complement, little-endian 2 byte integer                                |
+| 0x0D |    8 | Int8         | Two's complement, 1 byte integer                                              |
+| 0x0E |   64 | SplitIndex64 | Like Index64 but pages are stored in split + delta encoding                   |
+| 0x0F |   32 | SplitIndex32 | Like Index32 but pages are stored in split + delta encoding                   |
+| 0x10 |   64 | SplitReal64  | Like Real64 but in split encoding                                             |
+| 0x11 |   32 | SplitReal32  | Like Real32 but in split encoding                                             |
+| 0x12 |   16 | SplitReal16  | Like Real16 but in split encoding                                             |
+| 0x13 |   64 | SplitInt64   | Like Int64 but in split encoding                                              |
+| 0x14 |   32 | SplitInt32   | Like Int32 but in split encoding                                              |
+| 0x15 |   16 | SplitInt16   | Like Int16 but in split encoding                                              |
+
+Future versions of the file format may introduce addtional column types
+without changing the minimum version of the header.
+Old readers need to ignore these columns and fields constructed from such columns.
+Old readers can, however, figure out the number of elements stored in such unknown columns.
+
+The flags field can have one of the following bits set
+
+| Bit      | Meaning                                                      |
+|----------|--------------------------------------------------------------|
+| 0x01     | Elements in the column are sorted (monotonically increasing) |
+| 0x02     | Elements in the column are sorted (monotonically decreasing) |
+| 0x04     | Elements have only non-negative values                       |
+
+
+#### Alias columns
+
+An alias column has the following format
+
+```
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
++                      Physical Column ID                       +
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                           Field ID                            |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+The first 32bit integer references the physical column ID.
+The second 32bit integer references a field that needs to have the "alias field" flag set.
+The ID of the alias column itself is given implicitly by the serialization order.
+In particular, alias columns have larger IDs than physical columns.
```

Comment:
Can't you intertwine alias field and physical field?

Reply:
What is the purpose of having the alias column being on disk (since the alias field is already there)?

Reply:
What I mean is why is the alias represented, *on disk*, as a column rather than the column being recreated out of the alias field when reading the RNtuple meta-data?